lib/internal/Magento/Framework/Mview/etc/mview.xsd

```diff
@@ -46,6 +46,7 @@
         <xs:attribute name="id" type="xs:string" use="required" />
         <xs:attribute name="class" type="classType" use="required" />
         <xs:attribute name="group" type="xs:string" use="required" />
+        <xs:attribute name="walker" type="classType" default="Magento\Framework\Mview\View\ChangeLogBatchWalker" />
```

Comment:
Please, add in top of the file configuration with an example of usage of new features